index.html

```diff
@@ -0,0 +1,81 @@
+                                </div>
+                            </nav>
+                            </div>
+                            </header>
+                            <section>
+                                <div class="content-control">
+                                    <div class="left">
+                                    <h4>Facebook helps you connect and share with the people in your life.</h4>
+                                    <img src="img/world.jpg"/>
+                                    </div>
+                                    <div class="right">
+                                        <h1>Create an account</h1>
+                                        <p>It's free and always will be.</p>
+                                        <table>
+                                            <tr>
+                                                <td><input type="text" placeholder="First Name" class="ipt"> </td>
+                                                <td><input type="tex" placeholder="Last Name" class="ipt"> </td>
+                                            </tr>
+                                            <tr>
+                                                <td colspan="2"><input type="text" placeholder="Email" class="ipt2"></td>
+                                            </tr>
+                                            <tr>
+                                                <td colspan="2"><input type="text" placeholder="Re-enter email" class="ipt2"></td>
+                                            </tr>
+                                            <tr>
+                                                <td colspan="2"><input type="password" placeholder="Password" class="ipt2"></td>
+                                            </tr>
+                                            <tr>
+                                                <td><input type="datetime" placeholder="mm/dd/yyyy" class="ipt3"></td>
+                                            </tr>
+                                            </table>
+                                            <form>
+                                                <input type="radio" name="gender" value="male" class="ipt4">Male
+                                                <input type="radio" name="gender" value="Female" class="ipt4">Female
+                                            </form>
+                                            <form>
+                                            <div class="left2">
+                                                <h5>By clicking Create an account, you agree to our Terms and that you have<br>
+                                                read our Data Policy, including our Cookie Use.</h5>
+                                            </div>
+                                            <div class="login-button2">
+                                            <input type="submit" value="Create an account"><br><hr>
```

Comment:
Make the color of this button a little lighter. Light green or something. Compare with the prototype given